docs/12-docker-image.md

```diff
@@ -29,12 +29,28 @@ azb spark cluster create ... --docker-repo jiata/thunderbolt:0.1.0-spark1.6.3-py
 ## Using a custom Docker Image
 What if I wanted to use my own Docker image? _What if I want to use Spark v2.0.1 with Python v3.6.2?_
 
-You can build your own Docker image on top or beneath one of our supported base images _OR_ you can modify the [Azure Thunderbolt supported Dockerfile](../docker-image) and build your own image that way. 
+You can build your own Docker image on top or beneath one of our supported base images _OR_ you can modify the [Azure Thunderbolt supported Dockerfile](../docker-image) and build your own image that way.
 
 Please refer to ['../docker-image'](../docker-image) for more information on building your own image.
 
 Once you have your Docker image built and hosted publicly, you can then use the **--docker-repo** parameter in your **azb spark cluster create** command to point to it.
 
 ## Using a custom Docker Image that is Privately Hosted
 
-_Coming soon_
+To use a private docker image you will need to provide a docker username and password that have access to the repository you want to use.
+We also support custom docker endpoint(Not dockerhub)
```

Comment:
"(Not dockerhub)"

Do we explicitly not support dockerhub at the moment?

Reply:
Maybe this is a bit confusing. Meant you we also support a custom enpoint that is not dockerhub but its docker hub by default